src/js/storage/storage.js

```diff
@@ -0,0 +1,23 @@
+export const save = (key, value) => {
+  try {
+    const serializedState = JSON.stringify(value);
+    localStorage.setItem(key, serializedState);
+  } catch (error) {
+    console.error('Set state error: ', error.message);
+  }
+};
+export const load = key => {
+  try {
+    const serializedState = localStorage.getItem(key);
+    return serializedState === null ? undefined : JSON.parse(serializedState);
+  } catch (error) {
+    console.error('Get state error: ', error.message);
+  }
+};
+export const clear = key => {
+  try {
+    localStorage.removeItem(key);
+  } catch (error) {
+    console.error('Clear state error: ', error.message);
+  }
+};
```

Comment:
Nie widzę integracji zatem między tymi funkcjami, ale może to jest w późniejszych PR'kach.